storybook/src/pages/amsterdam/NavigationPage/NavigationPage.tsx

```diff
@@ -0,0 +1,22 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright Gemeente Amsterdam
+ */
+
+import { Breadcrumb, Grid } from '@amsterdam/design-system-react'
+import type { PropsWithChildren } from 'react'
+
+type NavigationPageProps = PropsWithChildren<{}>
+
+export const NavigationPage = ({ children }: NavigationPageProps) => (
+  <>
+    <Grid>
+      <Grid.Cell span={{ narrow: 4, medium: 6, wide: 7 }}>
+        <Breadcrumb>
+          <Breadcrumb.Link href="#">Home</Breadcrumb.Link>
+        </Breadcrumb>
+      </Grid.Cell>
+    </Grid>
+    <main id="main">{children}</main>
```

Comment:
We could simplify by using `<Grid as="main" id="main">` in children so you can have a flexible way to build `article` with `aside` or `section` layouts. 

```suggestion
    {children}
```

Reply:
Yes, good idea – done.